### DISCUSSION TITLE
Allow experimental features in REPL for non-snapshot releases

### DISCUSSION THREAD

**Discussion by prolativ** [scala/scala3#17843] [Feature Requests]
As REPL is used mainly for experimenting with the language and can't be used to produce binary artifacts, which could be used as dependencies in other projects, it seems reasonable to allow using experimental features in REPL with a non-snapshot version of the compiler (when enabled with an import or a compiler setting)

**Comment by dwijnand** [comment]
I was going to mention that the REPL is also very handy to experiment with the language as it is, i.e. without the experimental features interacting - but then that's already an issue if one uses a snapshot REPL...
